financial-connections/src/main/java/com/stripe/android/financialconnections/features/success/SuccessContent.kt

```diff
@@ -0,0 +1,223 @@
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.pluralStringResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.airbnb.mvrx.Async
+import com.airbnb.mvrx.Loading
+import com.stripe.android.financialconnections.R
+import com.stripe.android.financialconnections.features.common.V3LoadingSpinner
+import com.stripe.android.financialconnections.model.FinancialConnectionsSession
+import com.stripe.android.financialconnections.ui.TextResource
+import com.stripe.android.financialconnections.ui.components.FinancialConnectionsButton
+import com.stripe.android.financialconnections.ui.components.FinancialConnectionsScaffold
+import com.stripe.android.financialconnections.ui.components.FinancialConnectionsTopAppBar
+import com.stripe.android.financialconnections.ui.components.elevation
+import com.stripe.android.financialconnections.ui.theme.FinancialConnectionsTheme
+import kotlinx.coroutines.delay
+
+private const val ENTER_TRANSITION_DURATION_MS = 1000
+private const val CHECK_ALPHA_DURATION_MS = 250
+private const val SLIDE_IN_ANIMATION_FRACTION = 4
+
+@Composable
+internal fun SuccessContent(
+    overrideAnimationForPreview: Boolean,
+    completeSessionAsync: Async<FinancialConnectionsSession>,
+    payload: SuccessState.Payload,
+    onDoneClick: () -> Unit,
+    onCloseClick: () -> Unit,
+) {
+    val scrollState = rememberScrollState()
+    var showSpinner by remember { mutableStateOf(overrideAnimationForPreview.not()) }
+
+    if (payload.skipSuccessPane.not()) {
+        LaunchedEffect(true) {
+            delay(ENTER_TRANSITION_DURATION_MS.toLong())
+            showSpinner = false
+        }
+    }
+
+    FinancialConnectionsScaffold(
+        topBar = {
+            FinancialConnectionsTopAppBar(
+                showBack = false,
+                onCloseClick = onCloseClick,
+                elevation = scrollState.elevation
+            )
+        }
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(24.dp)
+        ) {
+            Box(
+                modifier = Modifier
+                    .weight(1f)
+                    .fillMaxWidth(),
+                contentAlignment = Alignment.Center
+            ) {
+                SpinnerToSuccessAnimation(
+                    customSuccessMessage = payload.customSuccessMessage,
+                    accountsCount = payload.accountsCount,
+                    showSpinner = showSpinner
+                )
+            }
+            SuccessFooter(
+                modifier = Modifier.alpha(if (showSpinner) 0f else 1f),
+                merchantName = payload.businessName,
+                loading = completeSessionAsync is Loading,
+                onDoneClick = onDoneClick
+            )
+        }
+    }
+}
+
+@Composable
+private fun SpinnerToSuccessAnimation(
```

Comment:
The animation feels a bit lifeless to me — it seems like it should be a bit more bouncy/celebratory.